benchmark_tests/test_extended_beam/test_moving_load_on_extended_track.py

```diff
@@ -0,0 +1,178 @@
+        NodalOutput.DISPLACEMENT_Y, NodalOutput.DISPLACEMENT_Z
+    ]
+    # Gauss point results
+    gauss_point_results = []
+
+    # Define the output process
+
+    vtk_output_process = Output(part_name="porous_computational_model_part",
+                                output_name="vtk_output",
+                                output_dir="output",
+                                output_parameters=VtkOutputParameters(file_format="ascii",
+                                                                      output_interval=10,
+                                                                      nodal_results=nodal_results,
+                                                                      gauss_point_results=gauss_point_results,
+                                                                      output_control_type="step"))
+    model.output_settings = [vtk_output_process]
+    # define json output parameters so that we can test that the fixities work
+    json_output_parameters = JsonOutputParameters(output_interval=0.5,
+                                                  nodal_results=nodal_results,
+                                                  gauss_point_results=gauss_point_results)
+    model.add_output_settings(json_output_parameters, f"soil_equivalent_rail_track_1", "output")
+    model.add_output_settings(json_output_parameters, f"constraint_soil_equivalent_rail_track_1", "output")
+
+    # Set mesh size
+    # --------------------------------
+    model.set_mesh_size(element_size=1)
+
+    input_folder = "benchmark_tests/test_extended_beam/inputs_kratos"
+
+    # Write KRATOS input files
+    # --------------------------------
+    stem = Stem(model, input_folder)
+    stem.write_all_input_files()
+
+    # Run Kratos calculation
+    # --------------------------------
+    stem.run_calculation()
+
+    expected_output_dir_temp = "benchmark_tests/test_extended_beam/inputs_kratos/output"
+    # check that the output is as expected (only the nodal displacements are checked)
+    with open(os.path.join(expected_output_dir_temp, "constraint_soil_equivalent_rail_track_1.json"), "r") as f:
+        constrain_output = json.load(f)
+    nodes = [item for name, item in constrain_output.items() if 'TIME' not in name]
+    assert np.all(np.array([node['DISPLACEMENT_X'] for node in nodes]) == 0)
+    assert np.all(np.array([node['DISPLACEMENT_Y'] for node in nodes]) == 0)
+    assert np.all(np.array([node['DISPLACEMENT_Z'] for node in nodes]) == 0)
+    with open(os.path.join(expected_output_dir_temp, "soil_equivalent_rail_track_1.json"), "r") as f:
+        soil_equivalent_output = json.load(f)
+    nodes = [item for name, item in soil_equivalent_output.items() if 'TIME' not in name]
+    assert np.all(np.array([node['DISPLACEMENT_X'] for node in nodes]) == 0)
+    assert np.any(np.array([node['DISPLACEMENT_Y'] for node in nodes]) != 0)
+    assert np.all(np.array([node['DISPLACEMENT_Z'] for node in nodes]) == 0)
```

Comment:
I write only a few coordinates of the part name soil_equivalent_rail_track_1 and constraint_soil_equivalent_rail_track_1. I will keep the check here since it is important for the issue. I also added a check for the VTK files